### ISSUE TITLE
Data structure for create_dataset.ipynb

### ISSUE THREAD

**Issue by Jonnyboyyyy** [ligerfotis/mvitac#6]
I think the structure of the folders after extracting from h5 files are not quite right.
The cell that removes objects that only has "success" or "failures" runs so that it removes all the data. 
I guess it's suppose to check if the name of the image file ends in success or failure. But it is implemented so that it's comparing in the folder where it has 3 folders = [gelsightA, gelsightB, kineticA_rgb].
This leads to erase all files since there is no name that ends in "success" or "failure"

**Comment by LeeYuanao** [comment]
> I think the structure of the folders after extracting from h5 files are not quite right. The cell that removes objects that only has "success" or "failures" runs so that it removes all the data. I guess it's suppose to check if the name of the image file ends in success or failure. But it is implemented so that it's comparing in the folder where it has 3 folders = [gelsightA, gelsightB, kineticA_rgb]. This leads to erase all files since there is no name that ends in "success" or "failure"

You are correct — with the current implementation all objects would indeed be deleted. In my view, since the final step already splits the dataset by object (object-wise split into train and test), whether a particular object only contains success or failure samples does not really matter anymore. Therefore, this cleaning step can safely be skipped.
